crates/solidity/inputs/language/src/definition.rs

```diff
@@ -3913,7 +3913,8 @@ codegen_language_macros::compile!(Language(
                                 Atom("\""),
                                 Atom("\\"),
                                 Atom("\n"),
-                                Atom("\r")
+                                Atom("\r"),
+                                Atom("\r\n")
```

Comment:
question: does this scanner always pick the longest match? if not, should `\r\n` be attempted first?